### ISSUE TITLE
UserWarning: Implicit dimension choice for log_softmax has been deprecated.

### ISSUE THREAD

**Issue by parkhy0106** [benatorc/OTGNN#2]
Initialized model with 13572 params
  0% 0/167 [00:00<?, ?it/s]/content/OTGNN/models/ot_modules.py:100: UserWarning: Implicit dimension choice for log_softmax has been deprecated. Change the call to include dim=X as an argument.
  nce_reg = torch.nn.LogSoftmax()(torch.stack(all_nce_dists))[0]

and the code was
  `....
         nce_reg = torch.nn.LogSoftmax()(torch.stack(all_nce_dists))[0]`

So I manually changed it as
  `....
         nce_reg = torch.nn.LogSoftmax(dim=0)(torch.stack(all_nce_dists))[0]`
Not to see the error message and it actually worked but I know that this wouldn't work well in some condition.
Is there any reason to choice dimension between -1 or 0? and what should I select? Anyway I love your contents!
